<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QLineEdit,QPlainTextEdit,QSpinBox,QDoubleSpinBox{

border:1px solid  #C0DCF2;
padding:2px;
background:none;
selection-background-color:#EEEEEE;
selection-color:#000000;
}

QLineEdit[echoMode=&quot;2&quot;]{
lineedit-password-character:9679;
}

QPushButton
{
    color: rgb(255, 255, 255);
    background-color: rgb(85, 170, 255);
    border:none;
    padding: 3px;
    font-family: &quot;微软雅黑&quot;;
    font-size: 12px;
    text-align: center;
}
QComboBox,QDateEdit,QSpinBox,QDoubleSpinBox{
border-radius:3px;
padding:3px 5px 3px 5px;
border:1px solid #C0DCF2;
}

QComboBox::drop-down,QDateEdit::drop-down{
subcontrol-origin:padding;
subcontrol-position:top right;
width:15px;
border-left-width:1px;
border-left-style:solid;
border-top-right-radius:3px;
border-bottom-right-radius:3px;
border-left-color:#C0DCF2;
}

QComboBox::down-arrow,QDateEdit::down-arrow{
image:url(:/image/array_down.png);
}
.QGroupBox{
border:1px solid #C0DCF2;
border-radius:5px;
font-family: &quot;微软雅黑&quot;;
}

QRadioButton{
color:#386487;
spacing:2px;
}

QRadioButton::indicator{
width:15px;
height:15px;
}

QRadioButton::indicator::unchecked{
image:url(&quot;D:/Documents/PyQt5/qt_protein/qt_protein/ui/images/radio_normal.png&quot;);
}

QRadioButton::indicator::checked{
image:url(&quot;D:/Documents/PyQt5/qt_protein/qt_protein/ui/images/radio_selected.png&quot;);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>73</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>变量个数</string>
    </property>
   </widget>
   <widget class="QComboBox" name="var_cunt_combox">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>70</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>381</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>通用属性</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>横轴上限</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="x_limit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>31</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>32</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>横轴下限</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="x_lower">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="y_limit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>72</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>72</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>纵轴下限</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="y_lower">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>纵轴上限</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>27</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>数据文件</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="file_path">
    <property name="geometry">
     <rect>
      <x>106</x>
      <y>22</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="open_file_button">
    <property name="geometry">
     <rect>
      <x>239</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>381</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>双变量（x为一变量，y为pcc，z为二变量）</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>75</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>x的取值个数</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="x_count">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>75</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>75</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>z的取值个数</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="z_count">
     <property name="geometry">
      <rect>
       <x>302</x>
       <y>74</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>横坐标</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>z坐标</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="x_coordinate_2">
     <property name="geometry">
      <rect>
       <x>77</x>
       <y>40</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="z_coordinate">
     <property name="geometry">
      <rect>
       <x>252</x>
       <y>40</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>381</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>单变量</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>258</x>
       <y>40</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>pcc</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="x_coordinate">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>44</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>45</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>纵坐标</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>45</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>横坐标</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="plot_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>529</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>绘制</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>424</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
